### PR TITLE
Fix tests and disable UI tests

### PR DESCRIPTION
**Pull request by morganchen12**
UI tests are currently broken due to underlying changes in Firebase Auth.

### CODE REVIEW

File: Auth/FirebaseAuthUI/FUIAuth.m [hunk 1]
```diff
@@ -197,9 +197,9 @@ - (void)signInWithProviderUI:(id<FUIAuthProvider>)providerUI
                                   credential:credential
                               resultCallback:result];
     } else {
-      [self.auth signInWithCredential:credential
-                           completion:^(FIRAuthDataResult *_Nullable authResult,
-                                        NSError *_Nullable error) {
+      [self.auth signInAndRetrieveDataWithCredential:credential
```
Comment (renkelvin) [inline Auth/FirebaseAuthUI/FUIAuth.m]: signInAndRetrieveDataWithCredential: is deprecated, see my reply in b/

File: FacebookAuth/FirebaseFacebookAuthUI.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -153,12 +153,20 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
+		7DC88936A359F0A20973DC4C /* Pods */ = {
```
Comment (renkelvin) [inline FacebookAuth/FirebaseFacebookAuthUI.xcodeproj/project.pbxproj]: remove?

File: Firestore/FirebaseFirestoreUI.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -75,6 +75,7 @@
 				8D69E46021DD8B2E00CFA49B /* FirebaseFirestoreUI */,
 				8D69E46B21DD8B2E00CFA49B /* FirebaseFirestoreUITests */,
 				8D69E45F21DD8B2E00CFA49B /* Products */,
+				905358392FD4EDB2D9802535 /* Pods */,
```
Comment (renkelvin) [inline Firestore/FirebaseFirestoreUI.xcodeproj/project.pbxproj]: remove?

File: Auth/FirebaseAuthUI/FUIAuth.m (hunk 1 above)
Comment (morganchen12) [reply Auth/FirebaseAuthUI/FUIAuth.m]: Will clean this up--I merged early so other PRs could have passing tests.

File: FacebookAuth/FirebaseFacebookAuthUI.xcodeproj/project.pbxproj (hunk 2 above)
Comment (morganchen12) [reply FacebookAuth/FirebaseFacebookAuthUI.xcodeproj/project.pbxproj]: These aren't removed by `pod deintegrate` for some reason. They're not harmful enough to warrant manually removing all of them.

File: Auth/FirebaseAuthUI/FUIAuth.m (hunk 1 above)
Comment (morganchen12) [reply Auth/FirebaseAuthUI/FUIAuth.m]: https://github.com/firebase/FirebaseUI-iOS/pull/690

**Pull request merged**